perimeter/shared_access.sp

```diff
@@ -895,3 +911,761 @@ control "rds_db_snapshot_shared_with_trusted_accounts" {
     service = "AWS/RDS"
   })
 }
+
+locals {
+  resource_policy_shared_account_sql = <<-EOT
+    select
+      r.__ARN_COLUMN__ as resource,
+      case
+        when jsonb_array_length(pa.allowed_principal_account_ids) = 0 then 'ok'
+        when pa.access_level = 'private' then 'ok'
+        when pa.access_level = 'public' then 'alarm'
+        when pa.allowed_principal_account_ids <@ to_jsonb(($1)::text[]) then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when jsonb_array_length(pa.allowed_principal_account_ids) = 0 then title || ' trust policy does not reference any accounts.'
+        when pa.access_level = 'private' then title || ' trust policy does not reference any cross accounts.'
+        when pa.access_level = 'public' then title || ' trust policy allows access to all AWS accounts.'
+        when pa.allowed_principal_account_ids <@ to_jsonb(($1)::text[]) then title || ' trust policy allows cross account access to trusted accounts.'
```

Comment:
```suggestion
        when pa.allowed_principal_account_ids <@ to_jsonb(($1)::text[]) then title || ' trust policy allows cross-account access from trusted accounts.'
```

```suggestion
        when pa.allowed_principal_account_ids <@ to_jsonb(($1)::text[]) then title || ' trust policy grants cross-account access to trusted accounts.'
```

I think wording wise, using `grants cross-account access to...` or `grants cross-account access from...` sounds better than `allows cross-account access to...`.

I think I like `allows cross-account access from...` better.